ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -37,33 +37,16 @@ import ErrorPanel from '../../../components/ErrorPanel';
 export default class ViewClients extends PureComponent {
   state = {
     clientSearch: '',
-    // eslint-disable-next-line react/no-unused-state
-    previousClientId: '',
     // This needs to be initially null in order for the defaultValue to work
     value: null,
```

Comment:
I'm seeing the following error in the console:

![Screen Shot 2019-04-29 at 8 52 01 AM](https://user-images.githubusercontent.com/3766511/56897344-4e050c80-6a5c-11e9-942f-d55b3d3690af.png)


To remove this error, I think we can do `value: undefined` instead. Let's also update the comment above it to say undefined instead of null :)